### DISCUSSION TITLE
GUI Rotation

### DISCUSSION THREAD

**Discussion by dk0297** [fhdm-dev/scoppy#131] [Q&A] [answered]
Hello,
on my phone(Samsung Galaxy A52s 5G / Android 13) the GUI does not rotate.
It would be incredibly helpful if the GUI could rotate into the second possible landscape mode if rotation is activated in Android.
The way my workspace is setup the oscilloscope would be upside-down.
Is it possible to rotate the GUI?

**Comment by fhdm-dev** [comment, marked as answer]
Yes. That turns out to be an easy fix. It will be in the next app update :smile:

**Reply by dk0297** [reply]
Sweet👍 Looking forward to the next update!
